docs/source/json-api/index.rst

```diff
@@ -316,9 +316,9 @@ For the Sandbox this corresponds to the ``--ledgerid MyLedger`` flag.
 
 .. note:: The value of ``applicationId`` will be used for commands submitted using that token.
 
-The value for ``actAs`` is specified as a list and you provide it with the party that you want to use.
-Such as the example which uses ``Alice`` for a party. Each request can only be for one party.
-For example you couldn't have ``actAs`` defined as ``["Alice", "Bob"]``.
+The value for ``actAs`` is specified as a list and you provide it with the party that you want to use,
+such as in the example above which uses ``Alice`` for a party. ``actAs`` may include more than just one party
+as the JSON API supports `multi-party submissions <https://blog.digitalasset.com/developers/roles-in-daml-introducing-multi-party-submissions>`_.
```

Comment:
@carrielaben-da I don't think that linking to the blog is a good idea as the URLs can change independently from the documentation. Thoughts? Do we have a page on the documentation we should either be pointing at? Or can we rather just not have a link here?

Reply:
I don't believe that we currently have a document that covers all of this in a unified fashion, although we touch on different aspects of it in several places. It's worth discussing whether we should and if so where that document should live. For right now, though, I think leaving out the link altogether is the best short-term answer.

Reply:
Link is removed.